From 74e2fe20d2aaa288a8447cfd2f5d5f0c1306ae75 Mon Sep 17 00:00:00 2001
From: Shawn Lin <shawn.lin@rock-chips.com>
Date: Wed, 21 Oct 2020 15:04:13 +0800
Subject: [PATCH] phy: add PHY_MODE_PCIE_BIFURCATION support

Bifurcation mode is tunable lanes which could be devided
into different individual combination.

Change-Id: Ie6a10694d3c4fd5da1a5673d58424de797f3622b
Signed-off-by: Shawn Lin <shawn.lin@rock-chips.com>
---
 include/linux/phy/phy.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/linux/phy/phy.h b/include/linux/phy/phy.h
index 54c574cc5217..b1cde581a130 100644
--- a/include/linux/phy/phy.h
+++ b/include/linux/phy/phy.h
@@ -45,6 +45,7 @@ enum phy_mode {
 	PHY_MODE_VIDEO_TTL,
 	PHY_MODE_PCIE_RC,
 	PHY_MODE_PCIE_EP,
+	PHY_MODE_PCIE_BIFURCATION,
 	PHY_MODE_GVI,
 	PHY_MODE_HDMI,
 };
-- 
2.35.3

